learn/on-premise/upgrade/run-patch.md

```diff
@@ -7,19 +7,39 @@ sidebar_label: "Run Patch"
 
 ## Running the Patch
 
+- When upgrading from 10.6.x to 10.7.x user need to download required content for migration process.for download migration tar please run the following command.
+
+```bash
+    wget https://s3.amazonaws.com/downloads.wavemaker.com/wme/wme.migration/wme_migrations.tar
+```
+
+- Extract wme_migrations.tar to /usr/local/content/wme/wme-installer/< version>/resources folder, for extract content run the following command
+
+```bash
+    sudo tar xvf wme_migrations.tar -C /usr/local/content/wme/wme-installer/<version>/resources/
+ ```
+
 - Run the bash script for upgrading to the desired version.
 
 ```shell
-bash /usr/local/content/wme/wme-installer/<version>/wme-installer.sh
+    bash /usr/local/content/wme/wme-installer/<version>/wme-installer.sh
 ```
 
+- To redirect HTTP traffics to HTTPS for accessing platform use enable-ssl option while running installer, to run with enable-ssl option please use following command.
```

Comment:
To make https work, You need to add SSL certificates to Platform  or router/dns level